tests/test_data/test_numbers.py

```diff
@@ -14,11 +14,11 @@ def test_floats(numbers):
 
 def test_primes(numbers):
     result = numbers.primes()
-    assert len(result) == 499
-    assert isinstance(result, array.array)
+    assert len(result) == 168
```

Comment:
This number seems like magic to me. Could you please explain what happens here?
A comment to some papers or articles would be awesome.

Reply:
Instead of checking whether every number in the returned list is a prime, this simply checks that the size of the list = number of primes in the given range (which is 1-999 by default).